code-challenges.test.js

```diff
@@ -60,8 +114,61 @@ var hand2 = [5, 5, 3, 3, 4]
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
-
-
-
+describe("When fullHouseChecker is called", () => {
+  it("return true, when an array of numbers is a full house",() => {
+    //arrange
+    const argument = hand1
+    //act
+    const actualResult = fullHouseChecker(argument)
+    //assert
+  expect(actualResult).toEqual(true)
+  })
+  it("return false when an array of number is not a full house",() => {
+    //arrange
+    const argument = hand2
+    //act
+    const actualResult = fullHouseChecker(argument)
+    //assert
+  expect(actualResult).toEqual(false)
+  })
+  it("return false when an array of number is not a full house",() => {
+    //arrange
+    const argument = hand3
+    //act
+    const actualResult = fullHouseChecker(argument)
+    //assert
+  expect(actualResult).toEqual(false)
+  })
+})
 
 // b) Create the function that makes the test pass.
+
+//pseudocode
+//create function that takes array as argument
+//.sort array in order, to create only two win conditions: [a,a,a,b,b] and [a,a,b,b,b]
+// use if else to check if meets either of these conditions
+//return true or false accordingly
+
+// const fullHouseChecker = (array) => {
+//   let sortedArray = array.sort((a,b) => (a-b))
+//   let [a,b,c,d,e] = sortedArray
+//   if(a === c && d === e && a !== e){
+//     return true
+//   }else if(a === b && c === e && a !== e){
+//     return true
+//   }else{
+//     return false
+//   }
+// }
+//
+
+
+//short method
+
+const fullHouseChecker = (array) => {
+  let sortedArray = array.sort((a,b) => (a-b))
+  let [a,b,c,d,e] = sortedArray
+  return ((a === c && d === e && a !== e) || (a === b && c === e && a !== e)) ? true:false
```

Comment:
```suggestion
  return ((a === c && d === e && a !== e) || (a === b && c === e && a !== e))
```
conditionals return booleans so they don't need to be provided explicitly